### PR TITLE
Rand uniform/normal

### LINKED ISSUE #155: [Feature] Add function generate data conform a known statistical distribution
This feature would be similar to either https://github.com/mrpowers-io/quinn/issues/88 or how sparklyr method to generate dataframe with certain distribution
E.x  https://spark.posit.co/packages/sparklyr/latest/reference/sdf_rexp.html

 Random integers generation:

- [ ]  Samples from ∼ U ( α , β ) #166 
- [ ]  Samples from ∼ N ( α , β ) #166 
- [x]  Samples from ∼ L ( α , β ) (Laplace distribution) #156 
- [x]  Samples from ∼ Γ ( α , β ) (Gamma distribution) #156

### PR DESCRIPTION
**Pull request by zeotuan**
Close #155  
- Add rand_range to Samples from ∼ U ( α , β )
- Add randn to Samples from ∼ N ( α , β )
- Rename randLaplace and randGamma to use snake case
- Add support for rand_laplace usage with mu and beta from column 
- More robust implementation of rand_laplace to deal with precision error

### CODE REVIEW

File: .github/workflows/core-ci.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        spark: ["3.0.1", "3.1.3", "3.2.4", "3.3.4"]
+        spark: ["3.0.1", "3.1.3", "3.2.4", "3.3.4", "3.4.3", "3.5.3"]
```
Comment (alfonsorr) [inline .github/workflows/core-ci.yml:14]: Can you move this update to other pr?

**Review by alfonsorr — CHANGES_REQUESTED** [review]

File: .github/workflows/core-ci.yml (hunk 1 above)
Comment (zeotuan) [reply .github/workflows/core-ci.yml:14]: I was hoping we could finish #165 first before this then I will rebase this on top after

Comment (alfonsorr) [reply .github/workflows/core-ci.yml:14]: Sure, but make a pr with the changes of the versions of spark and in the sbt file in a pr so the pr only contains the real content, then you can rebase both branches.

Comment (zeotuan) [reply .github/workflows/core-ci.yml]: I removed the unrelated change and cleaned the commit

**Pull request merged**